presto-docs/src/main/sphinx/admin/web-interface.rst

```diff
@@ -3,9 +3,20 @@ Web UI
 ======
 
 Presto provides a web-based user interface (UI) for monitoring a Presto cluster
-and managing queries. The web UI is accessible on the Presto coordinator via
+and managing queries. The Web UI is accessible on the coordinator via
```

Comment:
There are two other properties we may want to document: https://github.com/prestosql/presto/blob/master/presto-main/src/main/java/io/prestosql/server/ui/WebUiConfig.java#L49

This is the very technical details (which shouldn't be documented)

`web-ui.shared-secret` is used for signing the authentication cookie.  If this is not set the coordinator generates a random one, which means that when ever the coordinator is restarted, all users are logged out.  If set consistently, the user will stay logged in across coordinator restarts.

`web-ui.session-timeout` is how long a login token is valid.

Reply:
Maybe we should document them in the properties page and just link there

Reply:
I am moving all properties to that doc. Stay tuned for new commit .

Reply:
Available now.